day1/tests.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+from floor import Floor
+
+
+class FloorTest(unittest.TestCase):
+	def test_up_floor(self):
+		self.assertEqual(Floor().floor_position('('), 1)
+
+	def test_down_floor(self):
+		self.assertEqual(Floor().floor_position(')'), -1)
+
+	def test_up_down_floor(self):
+		self.assertEqual(Floor().floor_position(')())())'), -3)
+
+	def test_up_down_floor_input(self):
+		self.assertEqual(Floor().floor_position_input('input.txt'), 138)
```

Comment:
Estou em dúvida quanto a esse teste. Porque se algum dia eu quiser fazer o meu Advent Of Code em Python e quiser trapacear e usar o seu código para chegar ao resultado, eu vou lá e altero o input.txt para chegar ao resultado, mas o teste vai quebrar 😞 

Acho que o teste até é válido, para validar que ele está lendo um arquivo externo... Mas acho que não seria correto o mesmo arquivo que é usado para chegar no resultado... Fui claro? Faz sentido? 🤔 
Talvez seria mais válido se vc tivesse tipo um outro arquivo tipo `test/fixtures/input.txt` com algum outro valor qualquer e vc usasse ele para realizar seus testes.